test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -51,10 +51,17 @@
 	static Lookup lookup = MethodHandles.lookup();
 	static Class point2DClass = null;
 	static Class line2DClass = null;
+	static Class flattenedLine2DClass = null
 	static MethodHandle makePoint2D = null;
 	static MethodHandle makeLine2D = null;
+	static MethodHandle makeFlattenedLine2D = null
 	static MethodHandle getX = null;
 	static MethodHandle getY = null;
+	static MethodHandle getSt = null
```

Comment:
getFlatSt/WithFlatSt

Reply:
Should I also be replacing getEn/withEn with getFlatEn/WithFlatEn